konfera/models/ticket.py

```diff
@@ -29,6 +29,7 @@ class Ticket(KonferaModel):
     last_name = models.CharField(max_length=128)
     email = models.EmailField()
     phone = models.CharField(max_length=64, blank=True)
+    # TODO isn't the description useless? The general description of the ticket provides type.description...
```

Comment:
sorry for this "todo" comment, I left it as a remark for myself and forgot to discuss properly. 
We can perfectly let it called description unless users complain.